src/include/lib/simplehash.h

```diff
@@ -169,6 +170,14 @@ typedef struct SH_TYPE
 
 	/* user defined data, useful for callbacks */
 	void	   *private_data;
+
+	/*
+	 * number of times hash table is expanded
+	 *
+	 * Since the max size of hash table is UINT32_MAX, uint32 is good
+	 * enough for the number of expanded times.
+	 */
+	uint32 num_expansions;
```

Comment:
The new field may be put after `grow_threshold`, so it will be saved at the previous padding bytes.

Reply:
it's cherry-pick